### PR TITLE
adds 5 more medals

### PR DESCRIPTION
**Pull request by qwertyquerty**
because medals gud

### CODE REVIEW

**Comment by CthulhuOnIce** [comment]
Which metals?

**Comment by qwertyquerty** [comment]
```
#define MEDAL_APPLY_REAGENT_METH "NYOOOOM"
#define MEDAL_APPLY_REAGENT_SOYMILK "Soyboy"
#define MEDAL_15_AI_LAW_CHANGES "Dave, my mind is going"
#define MEDAL_DETONATE_WELDERBOMB "That was stupid of you."
#define MEDAL_SUCCUMB "Please just end the pain"
```

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm [hunk 1]
```diff
@@ -237,6 +237,14 @@
 	glass_name = "glass of soy milk"
 	glass_desc = "White and nutritious soy goodness!"
 
+//[BEGIN BEE EDIT]
+/datum/reagent/consumable/soymilk/on_mob_add(mob/living/L)
+	..()
+
+	if (L.client)
+		SSmedals.UnlockMedal(MEDAL_APPLY_REAGENT_SOYMILK,L.client)
+//[END BEE EDIT]
```
Comment (CthulhuOnIce) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm:246]: Add this to the beestation folder.

File: code/modules/reagents/chemistry/reagents/drug_reagents.dm [hunk 2]
```diff
@@ -179,6 +179,10 @@
 /datum/reagent/drug/methamphetamine/on_mob_add(mob/living/L)
 	..()
 	L.add_movespeed_modifier(id, update=TRUE, priority=100, multiplicative_slowdown=-2, blacklisted_movetypes=(FLYING|FLOATING))
+	//[BEGIN BEE EDIT]
+	if (L.client)
+		SSmedals.UnlockMedal(MEDAL_APPLY_REAGENT_METH,L.client)
+	//[END BEE EDIT]
```
Comment (CthulhuOnIce) [inline code/modules/reagents/chemistry/reagents/drug_reagents.dm:185]: this could also be added to beefolder

**Review by CthulhuOnIce — DISMISSED** [review]
A lot of this stuff could pretty easily be modularized, not even this stuff specifically

**Comment by qwertyquerty** [comment]
bruh moment

none of the other medals are modularized, trying to keep things consistent

**Comment by qwertyquerty** [comment]
I even put in the edit comments so you should lay off, let me be for just medals, I promise to modularize everything else but medals

**Comment by qwertyquerty** [comment]
like why have 2 of them in a certain file then all of the others in random files that's just a mess

**Comment by qwertyquerty** [comment]
consistency my man

**Pull request merged**